core/src/main/java/apoc/refactor/GraphRefactoring.java

```diff
@@ -68,10 +74,14 @@ public Stream<NodeRefactorResult> extractNode(@Name("relationships") Object rels
         return Util.relsStream(tx, rels).map((rel) -> {
             NodeRefactorResult result = new NodeRefactorResult(rel.getId());
             try {
-                Node copy = copyProperties(rel, tx.createNode(Util.labels(labels)));
-                copy.createRelationshipTo(rel.getEndNode(), RelationshipType.withName(outType));
+                Node copy = withTransactionAndRebind(db, tx, transaction -> {
+                    Node copyNode = copyProperties(rel, transaction.createNode(Util.labels(labels)));
+                    copyNode.createRelationshipTo(rel.getEndNode(), RelationshipType.withName(outType));
+                    return copyNode;
+                });
                 rel.getStartNode().createRelationshipTo(copy, RelationshipType.withName(inType));
                 rel.delete();
+                copy = Util.rebind(tx, copy);
```

Comment:
scusa perché fai ancora la rebind se già viene fatta sopra in `withTransactionAndRebind`?

Reply:
Sì giusto, non serve